templates/default/arguments.erb

```diff
@@ -127,6 +127,22 @@
 <% if node[:authconfig][:winbind][:join] && false -%>
 	--winbindjoin=<%= node[:authconfig][:winbind][:join] %>
 <% end -%>
+<% if node[:platform_version].to_i == 6 -%>
+<% if node[:authconfig][:ipav2][:enable] -%>
```

Comment:
indenting here does not match the rest of the file, if blocks that are inside other if blocks should be indented further.